options/options.js

```diff
@@ -0,0 +1,48 @@
+// Saves options to chrome.storage.sync.
+function save_options() {
+  var MB = document.getElementById('MB').value;
+  
+  chrome.storage.local.set({
+    MB: MB
+  }, function() {
+    // Update status to let user know options were saved.
+    var status = document.getElementById('status');
+    status.textContent = 'Options saved.';
+    setTimeout(function() {
+      status.textContent = '';
+    }, 1050);
+  });
+}
+
+// Restores option state using the preferences stored in chrome.storage.
+function restore_options() {
+    chrome.storage.local.get('MB', function(items) {
+        var MB;
+        if(items.MB !== undefined) {
+            MB = items.MB;
+        } else {
+            MB = "32";
```

Comment:
Can we declare this default in a constant at the top of the file instead? And it, too, should really be a number instead of a string.